src/go/rdctl/pkg/factoryreset/delete_data_linux.go

```diff
@@ -1,15 +1,17 @@
 package factoryreset
 
 import (
+	"context"
 	"os"
 	"path/filepath"
 
 	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/autostart"
 	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/paths"
+	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/process"
 	"github.com/sirupsen/logrus"
 )
 
-func DeleteData(appPaths paths.Paths, removeKubernetesCache bool) error {
+func DeleteData(ctx context.Context, appPaths paths.Paths, removeKubernetesCache bool) error {
```

Comment:
Why did you add the `ctx` argument? It seems to be unused on all 3 platforms.

Reply:
Mainly because having an unused context is easier to deal with than _not_ having it later when you need it. I guess I can remove it again now that it ends up unused…